packages/material-ui-styles/src/jssPreset.js

```diff
@@ -1,23 +1,25 @@
-import jssGlobal from 'jss-global';
-import jssNested from 'jss-nested';
-import jssCamelCase from 'jss-camel-case';
-import jssDefaultUnit from 'jss-default-unit';
-import jssVendorPrefixer from 'jss-vendor-prefixer';
-import jssPropsSort from 'jss-props-sort';
+import functions from 'jss-plugin-rule-value-function';
+import global from 'jss-plugin-global/dist/jss-plugin-global.cjs';
+import nested from 'jss-plugin-nested/dist/jss-plugin-nested.cjs';
```

Comment:
Is something broken with esm version?

Reply:
I have faced this exact issue: https://github.com/react-spring/react-spring/issues/278.

Reply:
Which exactly? I see people described a few different issues there.

Reply:
You are right, the issue mentioned two issues, one with the global polyfill (we are good on this e nd); the other one with the export syntax.

Reply:
Looking at the linked issue, it might also be an issue with Gatbsy but I haven't spent the time to dive. JSS v10, while in alpha, looks good to me from an "output" perspective. It's larger but faster. My main concern is with the upgrade path. We need to make sure it's working with no breaking change. I'm gonna try to scope the update to `@material-ui/styles`.